numpy/core/fromnumeric.py

```diff
@@ -2240,13 +2240,10 @@ def all(a, axis=None, out=None, keepdims=np._NoValue):
     >>> np.all([1.0, np.nan])
     True
 
-    >>> o=np.array([False])
+    >>> o=np.array(False)
     >>> z=np.all([-1, 4, 5], out=o)
-    Traceback (most recent call last):
-        ...
-    ValueError: output parameter for reduction operation logical_and has too many dimensions
-    >>> id(z), id(o), z                             # doctest: +SKIP
-    (28293632, 28293632, array([ True]))
+    >>> id(z), id(o), z
+    (28293632, 28293632, array([ True])) # may vary
```

Comment:
Is `z` really a 1d array here?

Reply:
No, it probably hasn't been tested in a long time because of the SKIP directive (and years without doctests, etc.) -- I've adjusted accordingly, though the proper result still won't be protected from regression with the in-line variable tag on the memory addresses.

Maybe better long-term to i.e., split on two separate lines, but I'll mark as resolved -- micro-improvement can always be made on a case-by-case basis in follow-ups.